searx/templates/simple/preferences.html

```diff
@@ -213,55 +280,51 @@ <h2>{{ _('Preferences') }}</h2>
     {{ tabs_close() }}
   {{ tab_footer() }}
 
-  {{ tab_header('maintab', 'ui', _('User interface')) }}
-  {% if 'locale' not in locked_preferences %}
-  <fieldset>
-    <legend>{{ _('Interface language') }}</legend>
-    <p class="value">
-      <select name='locale'>
-        {% for locale_id,locale_name in locales.items() | sort %}
-        <option value="{{ locale_id }}" {% if locale_id == current_locale %}selected="selected"{% endif %}>{{ locale_name }}</option>
-        {% endfor %}
-      </select>
-    </p>
-    <div class="description">{{ _('Change the language of the layout') }}</div>
-  </fieldset>
-  {% endif %}
-  {% if 'theme' not in locked_preferences %}
-  <fieldset>
-    <legend>{{ _('Themes') }}</legend>
-    <p class="value">
-      <select name="theme">
-        {%- for name in themes -%}
-        <option value="{{ name }}" {% if name == theme %}selected="selected"{% endif %}>{{ name }}</option>
-        {%- endfor -%}
-      </select>
-    </p>
-    <div class="description">{{ _('Change searx layout') }}</div>
-  </fieldset>
-  {% endif %}
-  {% if 'results_on_new_tab' not in locked_preferences %}
-  <fieldset>
-    <legend>{{ _('Results on new tabs') }}</legend>
-    <p class="value">
-      <select name='results_on_new_tab'>
-        <option value="1" {% if results_on_new_tab %}selected="selected"{% endif %}>{{ _('On') }}</option>
-        <option value="0" {% if not results_on_new_tab %}selected="selected"{% endif %}>{{ _('Off')}}</option>
-      </select>
-    </p>
-    <div class="description">{{_('Open result links on new browser tabs') }}</div>
-  </fieldset>
-  {% endif %}
-  {{ plugin_preferences('ui') }}
+  {{ tab_header('maintab', 'query', _('Special Queries')) }}
+    {% if answerers %}
+    <div class="scrollx">
+    <table class="striped">
+      <tr>
+        <th>{{ _('Allow') }}</th>
+        <th>{{ _('Keywords') }}</th>
+        <th>{{ _('Name') }}</th>
+        <th>{{ _('Description') }}</th>
+        <th>{{ _('Examples') }}</th>
+      </tr>
+      <td></td>
+      <th scope="colgroup" colspan="4">{{ _('This is the list of searx\'s instant answering modules.') }}</th>
+      {% for answerer in answerers %}
+      <tr>
+        <td></td>
+        <td>{{ answerer.keywords|join(', ') }}</td>
+        <td><div class="checkbox">{{ answerer.info.name }}</div></td>
```

Comment:
I just sued the checkbox class to make it look a bit better (with the margin); This seems a bit hacky tho!? Should I rather add an extra class to the css theme for this maybe?

Reply:
changed in 7cb5099

Reply:
I'm not sure if this is intended, but there is still the `checkbox` class.
![image](https://user-images.githubusercontent.com/1594191/134710003-26a42445-e2a4-4e77-98ea-71961f4625b0.png)

Without the labels don't overlap.

Reply:
Oh sorry missed that; Fixed in 3017227